src/libs/ValidationUtils.js

```diff
@@ -261,6 +261,15 @@ function isValidUSPhone(phoneNumber) {
     return CONST.REGEX.PHONE_E164_PLUS.test(phoneNumber.replace(CONST.REGEX.NON_ALPHA_NUMERIC, '')) && CONST.REGEX.US_PHONE.test(phoneNumber);
 }
 
+/**
+ * @param {String} phoneNumber
+ * @returns {Boolean}
+ */
+function isValidVBAPhone(phoneNumber) {
+    // Remove alphanumeric characters and validate that this is in fact a phone number
+    return CONST.REGEX.PHONE_E164_PLUS.test(phoneNumber.replace(CONST.REGEX.NON_ALPHA_NUMERIC, '')) && CONST.REGEX.PHONE_WITHOUT_COUNTRY_CODE.test(phoneNumber);
+}
+
```

Comment:
I am confused about this change. 
@puneetlath Could you please help me understand the phone number validations rules again? 
I want to know. 
- What chars are allowed in phone input? 
- Should this be the same for all inputs? But I see that a few components behave differently?
- What should be the correct validation rules for a phone number?

I don't have enough info to start a slack convo.

Reply:
@parasharrajat, this one is little different. We can't add any country code in it.

Reply:
Hm, interesting. So @thesahindia you're saying that on the VBA phone numbers the user isn't allowed to add a country code? Can you remind me why that is?

Reply:
@puneetlath, IIRC we only allow US numbers and I think we don't need to add country code for US numbers

Reply:
Ok got it. I would think we can just use the `isValidUSPhone` function. Even if they include a country code, I would think that's fine, as long as it is a valid US phone number. But if we're not sure if that's the case then let's confirm in Slack.

Reply:
@puneetlath, Waiting for the confirmation. 
https://expensify.slack.com/archives/C01GTK53T8Q/p1643738346932079

Reply:
As discussed on Slack: 
1. API for VBA only accepts 10 digit US number
2. We shouldn't bother the user with this restriction, so lets: 
a. Validate that the phone number is a valid us number (with or without country code)
b. Strip the country code if there is one before passing to the API

Reply:
So based on the conclusion @thesahindia you should use `isValidUSPhone` instead and remove this function. Right?

Reply:
@parasharrajat, isValidUSPhone function checks if the phone number has +1 in it. I have used isValidVBAPhone which skips that check and validate the numbers only